src/Listeners/ExceptionLogGroupCreatedListener.php

```diff
@@ -4,16 +4,23 @@
 
 use Illuminate\Support\Facades\Notification;
 use Taecontrol\MoonGuard\Repositories\UserRepository;
+use Taecontrol\MoonGuard\Notifications\SlackNotifiable;
 use Taecontrol\MoonGuard\Events\ExceptionLogGroupCreatedEvent;
 use Taecontrol\MoonGuard\Notifications\ExceptionLogGroupNotification;
 
 class ExceptionLogGroupCreatedListener
 {
     public function handle(ExceptionLogGroupCreatedEvent $event): void
     {
-        Notification::send(
-            UserRepository::all(),
-            new ExceptionLogGroupNotification($event->exceptionLogGroup)
-        );
+        $channels = config('moonguard.notifications.channels');
+
+        foreach ($channels as $channel) {
+            $users = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```

Comment:
Rename this variable into:

```suggestion
            $notifiables = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```